test/test_helper.exs

```diff
@@ -1,2 +1,36 @@
 ExUnit.start()
 Ecto.Adapters.SQL.Sandbox.mode(DigitalCollex.Repo, :manual)
+# Ensure that wallaby is started for functional test suites
+{:ok, _} = Application.ensure_all_started(:wallaby)
+Application.put_env(:wallaby, :base_url, DigitalCollexWeb.Endpoint.url)
+
+defmodule DigitalCollexWeb.FeatureCase do
```

Comment:
This needs to be moved to a separate file, however, I could not find a location by which Phoenix would automatically load this as a module (I attempted to use both `test/features/feature_case.exs` and `test/feature_case.exs`).